utils/log_writer.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import (
+	"bufio"
+	"io"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// NewLogWriter pipes stdout/err logs to logrus
+func NewLogWriter(fields log.Fields, level log.Level) io.WriteCloser {
```

Comment:
Fair point - I could swap the log levels. I was coming from the point of view that, Despite the name containing `err`, `stderr` is for debug statements (the ones you get when you `set -x`) and `stdout` is for stuff you care about generally.